Website/Portals/_default/HotcakesViews/_default/Scripts/Checkout.js

```diff
@@ -914,12 +913,9 @@
                                 }
                             });
                     })
-                    .fail(function (xhr, status, error) {
-                        console.log("xhr: " + xhr);
-                        console.log("status: " + status);
+                    .fail(function (error) {
                         console.log("error: " + error);
-                    })
-                    .always(function () { console.log("Error Always") });
```

Comment:
Why are we "always" logging this?  Shouldn't it be removed?

Reply:
This was removed.